Integrations/Cybereason/CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
 ## [Unreleased]
-
+Fixed an issue with the ***cybereason-query-file*** command.
```

Comment:
Maybe it is worthwhile to add a word on the issue? Say something like:  `Fixed an issue where  ***cybereason-query-file*** command did not pulled specific hashes`
Just so the customer would know what was the issue fixed.